From 7a9c36722511ce4df88b76cceceb241d6c6a151e Mon Sep 17 00:00:00 2001
From: Brian Norris <briannorris@chromium.org>
Date: Fri, 28 Feb 2020 15:50:47 -0800
Subject: [PATCH] DBus: Add "sae" to interface key_mgmt capabilities

This will be present when the driver supports SAE and it's included in
the wpa_supplicant build.

Signed-off-by: Brian Norris <briannorris@chromium.org>
---
 doc/dbus.doxygen                        | 2 +-
 wpa_supplicant/dbus/dbus_new_handlers.c | 6 ------
 2 files changed, 1 insertion(+), 7 deletions(-)

diff --git b/wpa_supplicant/dbus/dbus_new_handlers.c a/wpa_supplicant/dbus/dbus_new_handlers.c
index c842c50e9..55c5dbc99 100644
--- b/wpa_supplicant/dbus/dbus_new_handlers.c
+++ a/wpa_supplicant/dbus/dbus_new_handlers.c
@@ -2798,12 +2798,6 @@ dbus_bool_t wpas_dbus_getter_capabilities(
 			goto nomem;
 #endif /* CONFIG_WPS */
 
-#ifdef CONFIG_SAE
-		if ((capa.key_mgmt & WPA_DRIVER_CAPA_KEY_MGMT_SAE) &&
-		    !wpa_dbus_dict_string_array_add_element(&iter_array, "sae"))
-			goto nomem;
-#endif /* CONFIG_SAE */
-
 #ifdef CONFIG_OWE
 		if ((capa.key_mgmt & WPA_DRIVER_CAPA_KEY_MGMT_OWE) &&
 		    !wpa_dbus_dict_string_array_add_element(&iter_array, "owe"))
-- 
2.34.1

